source/blog/2015-06-12-ember-1-13-0-released.md

```diff
@@ -0,0 +1,578 @@
+
+The Glimmer rendering engine, [announced at EmberConf](https://www.youtube.com/watch?v=o12-90Dm-Qs&feature=youtu.be&t=47m21s) and introduced to Ember.js
+Canary on [May 5th](/blog/2015/05/05/glimmer-merging.html), is released today
+in Ember.js 1.13. Glimmer is the third overhaul of Ember's 1.x rendering
+layer (wow, right?), and dramatically improves rerender performance in many
+common scenarios. Additionally it lays important groundwork for the Ember 2.x
+development model.
+
+**We are tremendously excited to bring Glimmer to existing applications as
+a 1.x compatible minor release.** The herculean efforts of the Ember core team,
+addon authors, and community to achieve this release has been inspiring.
+Thank you! You are too many to possibly name.
+
+**1.13 also marks the final minor release of the Ember 1.x cycle.** As such,
+it contains a number of deprecations ahead of Ember 2.0. Resolving these
+deprecations in application code aligns that application with Ember's public
+API in 2.0. Any Ember app running 1.13 without causing a deprecation notice
+to fire should upgrade to 2.0 without any changes.
+
+#### Improved Rerender Performance
+
+Previous iterations of Ember's rendering layer were based on granular
+observation. Where a piece of dynamic content was rendered (such as a
+new array, or new component) the old DOM and possibly-invalid child views
+were discarded and new DOM and views created from scratch. Additionally,
+granular observation was computationally expensive to initialize and maintain
+for a large set of values.
+
+Glimmer replaces this approach with a value-diffing strategy.
+During render, each occurrence of dynamic content (for example a component,
+and each helper, or a data binding) generates a value representing its
+render state. Upon rerender, this tree of values is walked. Clean nodes are
+presumed to already be correctly rendered, dirtied nodes have their value
+recomputed and children values (which may or may not have dirtied) walked.
+
+The result is a stunning improvement in many rerender cases. This animated
+gif shows rerender performance of Ember 1.12 on the left, and Ember 1.13
+on the right:
+
+![](ANIMATED GIF)
+
+Thanks to [@wycats](https://twitter.com/wycats) and [@tomdale](https://twitter.com/tomdale)
+for their continued focus on improving
+Glimmer's performance, and to [LinkedIn](http://www.linkedin.com) and
+[Bustle](http://www.bustle.com/) for their
+generous sponsorship of this work.
+
+#### attrs, mut and auto-mut
+
+In Ember 1.13, components will now have an `attrs` property set with the
+passed values. For example:
+
+```hbs
+{{my-component name="Susan"}}
+```
+
+```js
+// app/components/my-component.js
+import Ember from "ember";
+
+export default Ember.Component.extend({
+  click() {
+    console.log(this.getAttr('name')); // -> logs "Susan"
+    console.log(this.attrs.name); // -> logs "Susan"
+    // The old behavior still works:
+    console.log(this.get('name')); // -> logs "Susan"
```

Comment:
It is not deprecated. I raised this in chat and there wasn't much feedback. I kind of wish it was deprecated, but it may make the migration path too difficult.